<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>575</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>41</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>215</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>232</red>
        <green>185</green>
        <blue>185</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>41</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>232</red>
        <green>185</green>
        <blue>185</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>41</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>232</red>
        <green>185</green>
        <blue>185</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>232</red>
        <green>185</green>
        <blue>185</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="cursor">
   <cursorShape>OpenHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Запустить сигнал</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить схему</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть схему</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать генератор</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>259</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>199</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../осциллограФ1.jpg</normaloff>../осциллограФ1.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Библиотека фильтров</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумматор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Библиотека генераторов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
